hawc/apps/lit/api.py

```diff
@@ -55,6 +56,27 @@ def reference_tags(self, request, pk):
         df = models.ReferenceTags.objects.as_dataframe(instance.id)
         return Response(df)
 
+    @decorators.detail_route(
+        methods=("get", "post"), url_path="references-download", renderer_classes=PandasRenderers
```

Comment:
this should just be get, no need to have post as well

Reply:
@rabstejnek after making this change, LGTM! Please make this change and then squash+merge. Congrats on your first PR 🎊

Reply:
Made the changes, will squash+merge now!